2020-09-20 00:47:28.3948||DEBUG|uccApiCore2.Program|init main |url: |action: 
2020-09-20 11:33:31.9296||ERROR|uccApiCore2.Controllers.TagController|Something went wrong inside TagController GetTag action: A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) |url: http://localhost/api/Tag/GetTag/|action: GetTag
2020-09-20 11:33:31.9296||ERROR|uccApiCore2.Controllers.FabricController|Something went wrong inside FabricController GetFabric action: A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) |url: http://localhost/api/Fabric/GetFabric/|action: GetFabric
2020-09-20 11:33:31.9296||ERROR|uccApiCore2.Controllers.CategoryController|Something went wrong inside CategoryController GetMainCategory action: A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) |url: http://localhost/api/Category/GetMainCategory/|action: GetMainCategory
2020-09-20 11:33:31.9296||ERROR|uccApiCore2.Controllers.CategoryController|Something went wrong inside CategoryController GetMainCategory action: A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) |url: http://localhost/api/Category/GetMainCategory/|action: GetMainCategory
2020-09-20 11:33:31.9296||ERROR|uccApiCore2.Controllers.SupplierController|Something went wrong inside SupplierController GetSupplier action: A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) |url: http://localhost/api/Supplier/GetSupplier/|action: GetSupplier
2020-09-20 11:33:31.9296||ERROR|uccApiCore2.Controllers.BrandController|Something went wrong inside BrandController GetAllBrand action: A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) |url: http://localhost/api/Brand/GetAllBrand/|action: GetAllBrand
2020-09-20 11:33:47.4106||ERROR|uccApiCore2.Controllers.ProductController|Something went wrong inside ProductController GetProductById action: A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) |url: http://localhost/api/Product/GetProductById/|action: GetProductById
2020-09-20 11:33:47.4106||ERROR|uccApiCore2.Controllers.CartController|Something went wrong inside CartController GetCartById action: One or more errors occurred. (A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)) |url: http://localhost/api/Cart/GetCartById|action: GetCartById
2020-09-20 11:33:47.4539||ERROR|uccApiCore2.Controllers.FabricController|Something went wrong inside FabricController GetAllFabricType action: A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) |url: http://localhost/api/Fabric/GetAllFabricType/|action: GetAllFabricType
2020-09-20 11:33:47.5566||ERROR|uccApiCore2.Controllers.CartController|Something went wrong inside CartController GetCartById action: One or more errors occurred. (A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)) |url: http://localhost/api/Cart/GetCartById|action: GetCartById
2020-09-20 11:33:47.6133||ERROR|Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware|An unhandled exception has occurred while executing the request System.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) ---> System.ComponentModel.Win32Exception (53): The network path was not found
   at uccApiCore2.Repository.LookupRepository.GetActiveColor() in C:\Project\Mohit\EcommApiCore\uccApiCore2.Repository\LookupRepository.cs:line 24
   at uccApiCore2.Controllers.LookupController.GetActiveColor() in C:\Project\Mohit\EcommApiCore\uccApiCore2\Controllers\LookupController.cs:line 29
   at lambda_method(Closure , Object )
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Cors.Infrastructure.CorsMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:53,State:0,Class:20|url: http://localhost/api/Lookup/GetActiveColor/|action: GetActiveColor
2020-09-20 11:33:47.6504||ERROR|Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware|An unhandled exception has occurred while executing the request System.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) ---> System.ComponentModel.Win32Exception (53): The network path was not found
   at uccApiCore2.Repository.LookupRepository.GetActiveSize() in C:\Project\Mohit\EcommApiCore\uccApiCore2.Repository\LookupRepository.cs:line 38
   at uccApiCore2.Controllers.LookupController.GetActiveSize() in C:\Project\Mohit\EcommApiCore\uccApiCore2\Controllers\LookupController.cs:line 35
   at lambda_method(Closure , Object )
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Cors.Infrastructure.CorsMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:53,State:0,Class:20|url: http://localhost/api/Lookup/GetActiveSize/|action: GetActiveSize
2020-09-20 11:34:28.8593||ERROR|uccApiCore2.Controllers.CartController|Something went wrong inside CartController GetCartById action: One or more errors occurred. (A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)) |url: http://localhost/api/Cart/GetCartById|action: GetCartById
2020-09-20 11:34:28.9080||ERROR|uccApiCore2.Controllers.CartController|Something went wrong inside CartController GetCartById action: One or more errors occurred. (A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)) |url: http://localhost/api/Cart/GetCartById|action: GetCartById
2020-09-20 11:35:02.5720||ERROR|uccApiCore2.Controllers.CartController|Something went wrong inside CartController GetCartById action: One or more errors occurred. (A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)) |url: http://localhost/api/Cart/GetCartById|action: GetCartById
2020-09-20 11:35:02.5885||ERROR|uccApiCore2.Controllers.CartController|Something went wrong inside CartController GetCartById action: One or more errors occurred. (A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)) |url: http://localhost/api/Cart/GetCartById|action: GetCartById
2020-09-20 14:02:23.2857||DEBUG|uccApiCore2.Program|init main |url: |action: 
2020-09-20 14:19:57.6222||DEBUG|uccApiCore2.Program|init main |url: |action: 
2020-09-20 15:45:28.1987||DEBUG|uccApiCore2.Program|init main |url: |action: 
2020-09-20 21:17:29.7343||DEBUG|uccApiCore2.Program|init main |url: |action: 
